src/Routes/Dashboard/SampleTabRoute.js

```diff
@@ -37,85 +84,118 @@ const SampleTabRoute = () => {
               }
             />
             <div className="pf-u-pl-3xl">
-              {[1, 2, 3].map((val) => (
-                <Switch
-                  className="pf-u-mt-md"
-                  key={val}
-                  id={`${val}-nested-switch`}
-                  aria-label={`${val}-nested-switch`}
-                  label={
-                    <Fragment>
-                      <Title headingLevel="h4" size="md">
-                        Nested switch {val}
-                      </Title>
-                      <TextContent>
-                        <Text component="small">
-                          Id culpa nostrud magna cupidatat commodo dolor tempor
-                          enim nisi irure duis sunt. Amet commodo dolore
-                          adipisicing velit aliquip est. Id laboris aute
-                          pariatur nulla. Fugiat qui exercitation ad sit. Dolore
-                          sit eiusmod et cupidatat qui cillum nulla pariatur
-                          consequat nulla irure deserunt incididunt esse.
-                        </Text>
-                      </TextContent>
-                    </Fragment>
-                  }
-                />
-              ))}
+              <Stack>
+                <StackItem>
+                  <Switch
+                    className="pf-u-mt-md"
+                    key="use-openscap"
+                    id="use-openscap"
+                    ouiaId="use-openscap"
+                    aria-label="Use OpenSCAP for Compliance policies"
+                    isChecked={useOpenSCAP}
+                    onChange={() => setUseOpenSCAP(!useOpenSCAP)}
+                    label={
+                      <Fragment>
+                        <Title headingLevel="h4" size="md">
+                          Use OpenSCAP for Compliance policies
+                          {getPopover()}
+                        </Title>
+                        <TextContent>
+                          <Text component="small">
+                            Required to use Compliance application
+                          </Text>
+                        </TextContent>
+                      </Fragment>
+                    }
+                  />
+                </StackItem>
+                <StackItem>
+                  <Switch
+                    className="pf-u-mt-md"
+                    key="use-resource-optimization"
+                    id="use-resource-optimization"
+                    ouiaId="use-resource-optimization"
+                    aria-label="Use Resource Optimization analysis"
+                    isChecked={useAnalysis}
+                    onChange={() => setUseAnalysis(!useAnalysis)}
+                    label={
+                      <Fragment>
+                        <Title headingLevel="h4" size="md">
+                          Use Resource Optimization analysis
+                          <Badge className="pf-u-ml-sm pf-u-mr-xs" isRead>
+                            Beta
```

Comment:
I know it is capitalized in the mocks, but should this be lowercase to match the beta badge in the masthead?

Reply:
i'd keep it as it is in the mocks. This is just filler content. We may remove the badge for the actual demo itself